lib/gitter-adapter.js

```diff
@@ -306,6 +307,19 @@ Adapter.prototype.listUsers = function(channel, key) {
   });
 };
 
+Adapter.prototype.autoJoin = function() {
+  this.gitterClient.currentUser()
+  .then(function(user) {
+    return user.rooms();
+  })
+  .then((function(rooms) {
```

Comment:
The extra parentheses are not needed and contradict the style already in place in this project.